### ISSUE TITLE
Commands in section 'Install from GitHub' won't install grav

### ISSUE THREAD

**Issue by georg-tiefenbrunn** [getgrav/grav-learn#415]
**Do you want to request a *feature* or report a *bug*?**
a *bug*

**What is the current behavior?**
Executing the commands listed in [Option 3: Install from GitHub](https://github.com/getgrav/grav-learn/blob/develop/pages/01.basics/03.installation/docs.md#option-3-install-from-github) won't install grav.

```bash
$ cd ~/webroot
$ git clone -b master https://github.com/getgrav/grav.git
```
This will clone grav into `~/webroot/grav` thus executing `composer install --no-dev -o` won't install anything (*Composer could not find a composer.json file in ~/webroot*).

To correctly install grav in subfolder `grav` the line `$ cd ~/webroot/grav` from step 3 (Install ... using the Grav CLI application bin/grav) should be moved to step 2 (Install vendor dependencies via composer) before running `composer install`.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

Just run the commands as listed in the documentation.

**What is the expected behavior?**
Grav should be installed and running after executing the commands from the docs.

**Which versions of OS are affected by this issue? Did this work in previous versions?**
git version 2.1.4

**Comment by flaviocopes** [comment]
That's right. You can directly edit the page from https://github.com/getgrav/grav-learn/edit/develop/pages/01.basics/03.installation/docs.md and submit the change to be included.
